packages/frontend/core/src/modules/collection-rules/impls/order-by/checkbox.ts

```
@@ -0,0 +1,38 @@
+    _items$: Observable<Set<string>>,
+    params: OrderByParams
+  ): Observable<string[]> {
+    const isDesc = params.desc;
+    return combineLatest([
+      this.docsService.list.docs$, // We need the complete doc list as docs without property values should default to false
+      this.docsService.propertyValues$('custom:' + params.key),
+    ]).pipe(
+      map(([docs, values]) => {
+        const result: [string, boolean][] = [];
+        for (const doc of docs) {
+          const value = values.get(doc.id) === 'true' ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)